docs/install_ovos_core.md

```diff
@@ -1,70 +1,185 @@
+
+**ovos-messagebus**
+
+`pip install --pre ovos-messagebus`
+
+[Starting Modules - messagebus](starting_modules.md#messagebus)
+
+**ovos-audio**
+
+`pip install --pre ovos-audio`
+
+[Starting Modules - audio](starting_modules.md#audio)
+
+**dinkum-listener**
+
+`pip install --pre ovos-dinkum-listener`
+
+[Starting Modules - listener](starting_modules.md#listener)
+
+**ovos-phal**
+
+`pip install --pre ovos-phal`
+
+[Starting Modules - phal](starting_modules.md#phal)
+
+## From Source
+We will use git to clone the repositories to a local directory.  While not specificly necessasary, we are assuming this to be the users `HOME` directory.
+
+### ovos-core
+Install `ovos-core` from github source files
+
+`git clone https://github.com/OpenVoiceOS/ovos-core`
+
+The ovos-core repository provides `extra` requirements files.  For the complete stack, we will use the `mycroft.txt` file.
+
+`pip install ~/ovos-core[mycroft]`
+
+This should install everything needed to use the basic OVOS software stack.
+
+**NOTE** this also installs `lgpl` licenced software
+
+[Starting OVOS](starting_intro.md)
+
+### Install individual modules
+
+Some systems may not require a full install of OVOS.  Luckly, it can be installed as individual modules.
+
+**core library**
+
+`git clone https://github.com/OpenVoiceOS/ovos-core`
+
+`pip install ~/ovos-core`
+
+This is the minimal library needed as the brain of the system.  There are no skills, no messagebus, and no plugins installed yet
+
+[Starting Core](starting_modules.md#core)
+
+**messagebus**
+
+`git clone https://github.com/OpenVoiceOS/ovos-messagebus`
+
+`pip install ~/ovos-messagebus`
+
+This is the nervous system of OVOS needed for modules to talk to eachother
+
+[Starting the Messagebus](starting_modules.md#messagebus)
+
+**listener**
+
+OVOS has updated thier listener to use `ovos-dinkum-listener` instead of `ovos-listener`.  It is code from [mycroft-dinkum](https://github.com/MycroftAI/mycroft-dinkum) adopted for use with the OVOS ecosystem.  Previous listeners are still available, but not recommended.
+
+`git clone https://github.com/OpenVoiceOS/ovos-dinkum-listener`
+
+`pip install ~/ovos-dinkum-listener`
+
+You now have what is needed for OVOS to use a microphone and its associated services.  WakeWords, HotWords, and STT
+
+[Starting the listener](starting_modules.md#listener)
+
+**PHAL**
+
+The OVOS Plugin based Hardware Abstraction Layer is what is used to allow the OVOS software to communicate with hardware devices such as the [OS system](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-system) or interacting with the [Mycroft Mark1 device](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-mk1)
+
+The PHAL system consists of two interfaces.
+
+`ovos-phal` is the basic interface that normal plugins would use.
+
+`ovos-admin-phal` is used where superuser privleges are needed.
+
+**Be extreamly careful when installing `admin-phal` plugins as it gives full control over the host system**
```

Comment:
```suggestion
**Be extremely careful when installing `admin-phal` plugins as it gives full control over the host system**
```

Reply:
```suggestion
**Be extremely careful when installing `admin-phal` plugins as they provide full control over the host system.**
```

Seems you forgot to add a period, @mikejgray. I rephrased the sentence because @builderjer refers to “`admin-phal`” as “plugin**s**”, consequently denoting that `admin-phal` is an abstraction of multiple packages, so the subsequent pronoun should reflect that. I can't actually verify whether this is true, however.